cmd/getenvoy/main.go

```diff
@@ -15,14 +15,36 @@
 package main
 
 import (
+	"fmt"
+	"io"
 	"os"
 
 	cmdutil "github.com/tetratelabs/getenvoy/pkg/cmd"
 	"github.com/tetratelabs/getenvoy/pkg/globals"
 )
 
 func main() {
-	if cmdutil.Execute(cmdutil.NewRoot(&globals.GlobalOpts{})) != nil {
-		os.Exit(1)
+	os.Exit(run(os.Stdout, os.Stderr, os.Args))
+}
+
+// run handles all error logging and coding so that no other place needs to.
+func run(stdout, stderr io.Writer, args []string) int {
```

Comment:
I moved the error printing logic here so that we don't need to test stderr formatting of an error on every test. This was distracting, spread logic in different places and generally made it harder to get good test coverage.